get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
```

Comment:
We can leave it as it is in `develop`.

Doing this change simply allows for `json` or `js` configs and matched how the webpack config works.